mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/processing/impl/FunctionBridgeValueProcessor.java

```diff
@@ -19,21 +19,21 @@
 public class FunctionBridgeValueProcessor<T, R> implements ValueProcessor {
 
 	private final FunctionBridge<T, R> bridge;
-	private final BridgedElementReader<? extends T> bridgedElementReader;
+	private final BridgedElementAccessor<? extends T> bridgedElementAccessor;
```

Comment:
You don't talk about this change in the commit summary. What was the issue with reader? If we can only read the value, I think it's a better name.

Reply:
Right, I didn't explain it...
For `IndexFieldAccessor`, the reason is we will need both write (when indexing) and read (when projecting).
For `BridgedElementAccessor`, I must admit I don't remember in details (it's been a while...) but I think the idea was to shield ourselves from a future need to refer to a property in a context where "reading" is not really relevant. There is for example reflection (gathering some metadata through the accessor). We can also reasonably assume that we will need to refer to properties when registering things in `SearchMetadata` for instance, for instance to declare a projection on this or that entity property (as opposed to index field).
I don't have a strong case for Accessor, I'll give you that. It's more that once we go for Reader, then it only makes sense to add reading-related features, and anything else will not fit well.